JavaDirectory/src/test/java/com/bug/free/invention/api/seleniumTests/jobRolesPageTest.java

```diff
@@ -0,0 +1,62 @@
+package com.bug.free.invention.api.seleniumTests;
+import com.bug.free.invention.api.framework.FunctionalTest;
+import com.bug.free.invention.api.pages.jobRolesPage;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class jobRolesPageTest extends FunctionalTest {
+
+    @Test
+    public void getJobRolesTableTest() {
+        driver.get("http://localhost:6555/JobsTable");
+
+        jobRolesPage jobRolesPage = new jobRolesPage(driver);
+
+        assertTrue(jobRolesPage.isInitialised());
+    }
+
+    @Test
+    public void jobRolesJobTitleCellIsNotNull(){
+        driver.get("http://localhost:6555/JobsTable");
+
+        jobRolesPage jobRolesPage = new jobRolesPage(driver);
+
+        assertNotNull("This is job title cell is null",jobRolesPage.getJobTitleCellText());
```

Comment:
The message here reads strangely. Did you mean something like the following?
```suggestion
        assertNotNull("The job title cell is null",jobRolesPage.getJobTitleCellText());
```

Similarly for other tests here